### ISSUE TITLE
Building queue with one build-task (Feature request)

### ISSUE THREAD

**Issue by john-haugland** [equinor/radix-platform#257]
I would be be nice if the building queue only allowed for one build task. When a building job takes long time ( e.g. 30-60 min ) several commits or merges to the master branch can easily fill up with lots of tasks when it is only the last one that we want to build. WebAssembly code from C++ is a process that is really slow also on my computer when running locally.  

Application name in Radix: 
ipvp_apps. Currently the build time is about 30 minutes.

Application component: 
Emscripten to make WebAssembly code from C++.

**Comment by keaaa** [comment]
Thanks for your input. Would a "abandon job" feature solve your need?

**Comment by john-haugland** [comment]
Hi. Yes that could solve the issue. I.e. when several build-tasks have queued up. You would first manually remove all old jobs so that only one job is left and then abandon the ongoing job. Feasible ? Would be nice to have a button to do this in a single operation on the web console.

**Comment by keaaa** [comment]
Hi, we have added a "Stop Job" feature in Radix - It can be used on ongoing and queued jobs. Please report back if this feature solves your need, or if there are other things thats needed before we can close this issue.

**Comment by keaaa** [comment]
Since I have not heard back from you, im closing the issue.

**Comment by john-haugland** [comment]
Thanks!
